.github/workflows/pr_diff.yaml

```diff
@@ -0,0 +1,52 @@
+name: PR Diff for Helm chart
+on:
+  pull_request:
+  pull_request_target:
+  
+permissions:
+  pull-requests: write
```

Comment:
I think this is the crux. From their [docs](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs), all other permissions should flip to none as per this line. If an attacker would remove this one, in order to gain full permissions from `pull_request_target`, his attack would fail because changes to `on:` on `permissions:` 
blocks should only go into effect already on the PR if the change is not coming from a fork. This `on:` block is already verfied as per this PR (I can't elevate). The first part, so I expect similar behaviour for the `permissions:` block.

More info on the triggers, here in context of a `docker build --push`, where the push should fail in PRs coming from the fork (no access to Github Secrets): https://github.com/pelias/docker/issues/264#issuecomment-1212424894

Reply:
Alright. There aren't currently any secrets on this repo, since there's no third-party service involved in CI nor automated release/deployment stages. Limiting the permissions to `pull-requests: write` seems reasonable, although still seems potentially broad (i.e: does it mean the action can write to *any* PR against the target?). It's not a significant enough risk, to me, to block from merging, though. Let's try it.

Reply:
> does it mean the action can write to any PR against the target?

I wished GH docs were more elaborate here...